src/main/java/com/iexec/common/replicate/ReplicateStatusDetails.java

```diff
@@ -29,23 +29,32 @@
 public class ReplicateStatusDetails {
 
     public static final int MAX_STDOUT_LENGTH = 100000;
+    public static final int MAX_STDERR_LENGTH = 100000;
```

Comment:
Using a single `MAX_STD_LENGTH`, should be fine, what do you think?
If you agree, you can use a single method for substring stds

Reply:
Sure, we can do so. Please see https://github.com/iExecBlockchainComputing/iexec-common/pull/326/commits/56f5f3b8b7df6dcd848759bba01d0fcd0325d93e.